scripts/cob-command

```diff
@@ -148,6 +148,9 @@ stop_session(){
     if [[ $command == *'roscore'* ]]; then
```

Comment:
I guess you have to add something with `$lock_slack` to the if condition in line 147 (one line above)